.github/workflows/dispatch-export.yml

```
@@ -36,7 +39,7 @@ jobs:
     runs-on: ubuntu-24.04
     if: ${{ github.event.inputs.export_platform == 'windows' || github.event.inputs.export_platform == 'all' }}
     container:
-      image: barichello/godot-ci:4.5
+      image: barichello/godot-ci:${{ env.GODOT_VERSION }}
```

actionlint (1.7.9)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)